#include "app.h"

#include <string>
#include <iomanip>
#include <iostream>
#include <exception>

int main()
{
	/*std::unique_ptr<app::App> app;
	
	try {
		app = std::make_unique<app::App>();
		app->mainloop();
	}
	catch (std::runtime_error e) {
		std::cerr << "Runtime error : " << e.what() << std::endl;
	}
	catch (...) {
		std::cerr << "Unexpected error." << std::endl;
		std::terminate();
	}*/

	return 0;
}